Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'coffee-rails', '~> 4.0.0'
 gem 'jquery-rails'
 gem 'jbuilder', '~> 2.0'
 
-gem 'comfortable_mexican_sofa', '~>2.0.1', git: 'git@github.com:moneyadviceservice/comfortable-mexican-sofa.git'
+gem 'comfortable_mexican_sofa', '~>2.0.1', path: '../comfortable-mexican-sofa'# git: 'git@github.com:moneyadviceservice/comfortable-mexican-sofa.git'
```

Comment:
Actually, no. I will merge the comfy PR first and then address the proper version on the Gemfile, so this line could be ignored until I merge the PR and just link the version. I added before the explanation on the PR description about it,  perhaps I should have put more clearer.